quarkchain/p2p/p2p_server.py

```diff
@@ -158,10 +158,17 @@ def _get_discv5_topic(self) -> bytes:
             await self._receive_handshake(reader, writer)
         except expected_exceptions as e:
             self.logger.debug("Could not complete handshake: %s", e)
+            self.logger.error("Could not complete handshake: %s", e)
```

Comment:
duplicate line

Reply:
they are not duplicate, but I will remove the error log